src/styles/index.scss

```diff
@@ -1,3 +1,138 @@
+$button-color:#00acdc;
+$second-text:#616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background: rgb(213, 213, 209);
+}
+
+$star-size: 16px;
+
+.stars {
```

Comment:
just as recommendation. 
It'd nice if you separate styles at different scss files (like `variables.scss`, `stars.scss`, `card.scss`, `common.scss`)